find_missing.py

```
@@ -0,0 +1,24 @@
+import xml.etree.ElementTree as ET
+import os
+
+def get_strings(path):
+    try:
+        tree = ET.parse(path)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)